PVAO.Web/Views/Home/Index.cshtml

```diff
@@ -1,23 +1,27 @@
 ﻿@{
     ViewBag.Title = "Home Page";
 }
+@section styles {
+    <style>
+        h1 {
+            text-align: center;
+            color: #543fff;
+        }
+    </style>
+}
 
 <div class="dash-content spacer">
     <div class="container-fluid">
         <div class="row">
-            <div class="col-md-12">
-                <div class="recent-activity">
-                    <div class="act-title">
-                        <h5><span id="user-greetings"></span></h5>
-                    </div>
-                    <ul class="act-wrap">
-                        <li class="alert br-o fade show">
-                            Welcome to <span class="review-stat">Philippine Veterans Affairs Office</span> web portal!
-                            <p>Get started by opening a dashboard, over remittance, veterans or beneficiary list.</p>
-                        </li>
-                    </ul>
+            <div class="col-sm-6">
+                <div class="jumbotron text-center">
+                    <h1>Welcome to<br/>PVAO Portal</h1>
+                    <p>Get familiar with the dashboard, here are some ways to get started.</p>
```

Comment:
Change the text to "Get started by opening a dashboard, over-remittance, accrued or additional funded cheques."